### DISCUSSION TITLE
Recovering projects and avoiding software crashes with large amount of data

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#1535] [Questions & Problems]
Dear community,

I am facing difficulties while working with a large amount of observed data (approximately 100 C-t profiles) in MoBi's Parameter Identification function. The software crashes frequently, either during the run itself or when I attempt to add more observed data to the same PI step.

I tried increasing the value of GDIProcessHandleQuota, as suggested in one of the discussions, but MoBi still crashes. Moreover, when I tried to reopen the last saved workspace, I encountered the following error. Even after restarting both my PC and the project, I cannot locate any of the building blocks, simulations, or parameter identification steps that I had created.

Is there a way to recover this project and prevent future crashes?

Best regards,
Aya

Project interface:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/67606842/e4cb3946-8a02-4842-b1b8-b2b5eb8b4522)

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/67606842/8cdcb901-6305-4ade-94e8-4cde64bfe9c4)

Error:
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

***** Exception Text *****
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
   at System.Collections.ArrayList.get_Item(Int32 index)
   at DevExpress.XtraBars.Docking.ReadOnlyPanelCollection.get_Item(Int32 index)
   at DevExpress.XtraBars.Docking.DockManager.CheckActivePanel()
   at DevExpress.XtraBars.Docking.DockManager.DeserializeLayoutManager()
   at DevExpress.XtraBars.Docking.DockManager.SafeOnLoaded()
   at DevExpress.XtraBars.Docking.DockManager.OnLoaded()
   at DevExpress.XtraBars.Docking.DockManager.DevExpress.Utils.Serializing.IXtraSerializable.OnEndDeserializing(String restoredVersion)
   at DevExpress.Utils.Serializing.Helpers.DeserializeHelper.RaiseEndDeserializing(Object obj, String restoredLayoutVersion)
   at DevExpress.Utils.Serializing.Helpers.DeserializeHelper.DeserializeObject(Object obj, IXtraPropertyCollection store, XtraSerializationFlags parentFlags, OptionsLayoutBase options)
   at DevExpress.Utils.Serializing.XtraSerializer.DeserializeObject(Object obj, IXtraPropertyCollection store, OptionsLayoutBase options)
   at DevExpress.XtraBars.Docking.DockManager.RestoreLayoutCore(XtraSerializer serializer, Object path)
   at MoBi.UI.Settings.UserSettings.RestoreLayout()
   at MoBi.Presentation.Presenter.MoBiMainViewPresenter.RestoreLayout()
   at MoBi.UI.Views.MoBiMainView.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at DevExpress.XtraEditors.XtraForm.WndProc(Message& msg)
   at DevExpress.XtraBars.Ribbon.RibbonForm.WndProc(Message& msg)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

***** Loaded Assemblies *****
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4614.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
MoBi
    Assembly Version: 11.2.138.0
    Win32 Version: 11.2.138
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/MoBi.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4550.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4614.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4390.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
DevExpress.Utils.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.Utils.v21.2.DLL
----------------------------------------
DevExpress.Data.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.Data.v21.2.DLL
----------------------------------------
OSPSuite.Utility
    Assembly Version: 4.0.0.4
    Win32 Version: 4.0.0.4
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Utility.DLL
----------------------------------------
netstandard
    Assembly Version: 2.0.0.0
    Win32 Version: 4.8.3761.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/netstandard/v4.0_2.0.0.0__cc7b13ffcd2ddd51/netstandard.dll
----------------------------------------
MoBi.UI
    Assembly Version: 11.2.138.0
    Win32 Version: 11.2.138
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/MoBi.UI.DLL
----------------------------------------
Microsoft.VisualBasic
    Assembly Version: 10.0.0.0
    Win32 Version: 14.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Microsoft.VisualBasic/v4.0_10.0.0.0__b03f5f7f11d50a3a/Microsoft.VisualBasic.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4614.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
OSPSuite.Core
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Core.DLL
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4190.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
DevExpress.Data.Desktop.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.Data.Desktop.v21.2.DLL
----------------------------------------
OSPSuite.Infrastructure.Castle
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Infrastructure.Castle.DLL
----------------------------------------
Castle.Windsor
    Assembly Version: 5.0.0.0
    Win32 Version: 5.0.1
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Castle.Windsor.DLL
----------------------------------------
OSPSuite.Presentation
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Presentation.DLL
----------------------------------------
Microsoft.Extensions.Logging.Abstractions
    Assembly Version: 3.1.6.0
    Win32 Version: 3.100.620.31605
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Microsoft.Extensions.Logging.Abstractions.DLL
----------------------------------------
MoBi.Core
    Assembly Version: 11.2.138.0
    Win32 Version: 11.2.138
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/MoBi.Core.DLL
----------------------------------------
MoBi.Presentation
    Assembly Version: 11.2.138.0
    Win32 Version: 11.2.138
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/MoBi.Presentation.DLL
----------------------------------------
OSPSuite.UI
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.UI.DLL
----------------------------------------
OSPSuite.Assets.Images
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Assets.Images.DLL
----------------------------------------
OSPSuite.Assets
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Assets.DLL
----------------------------------------
Northwoods.Go
    Assembly Version: 5.2.0.46
    Win32 Version: 5.2.0.46
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Northwoods.Go.DLL
----------------------------------------
Castle.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.3.1
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Castle.Core.DLL
----------------------------------------
Anonymously Hosted DynamicMethods Assembly
    Assembly Version: 0.0.0.0
    Win32 Version: 4.8.4614.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/mscorlib/v4.0_4.0.0.0__b77a5c561934e089/mscorlib.dll
----------------------------------------
System.ValueTuple
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.ValueTuple/v4.0_4.0.0.0__cc7b13ffcd2ddd51/System.ValueTuple.dll
----------------------------------------
OSPSuite.Infrastructure.Import
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Infrastructure.Import.DLL
----------------------------------------
System.Drawing.Common
    Assembly Version: 4.0.0.1
    Win32 Version: 4.6.26919.02
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/System.Drawing.Common.DLL
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4585.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------
DevExpress.RichEdit.v21.2.Core
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.RichEdit.v21.2.Core.DLL
----------------------------------------
DevExpress.Office.v21.2.Core
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.Office.v21.2.Core.DLL
----------------------------------------
DevExpress.XtraBars.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraBars.v21.2.DLL
----------------------------------------
DevExpress.XtraEditors.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraEditors.v21.2.DLL
----------------------------------------
DevExpress.XtraRichEdit.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraRichEdit.v21.2.DLL
----------------------------------------
DevExpress.XtraGrid.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraGrid.v21.2.DLL
----------------------------------------
DevExpress.XtraTreeList.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraTreeList.v21.2.DLL
----------------------------------------
Northwoods.Go.Layout
    Assembly Version: 5.2.0.46
    Win32 Version: 5.2.0.46
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Northwoods.Go.Layout.DLL
----------------------------------------
DevExpress.XtraPivotGrid.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraPivotGrid.v21.2.DLL
----------------------------------------
DevExpress.PivotGrid.v21.2.Core
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.PivotGrid.v21.2.Core.DLL
----------------------------------------
DevExpress.XtraCharts.v21.2.UI
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraCharts.v21.2.UI.DLL
----------------------------------------
DevExpress.XtraCharts.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraCharts.v21.2.DLL
----------------------------------------
DevExpress.Charts.v21.2.Core
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.Charts.v21.2.Core.DLL
----------------------------------------
DevExpress.Printing.v21.2.Core
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.Printing.v21.2.Core.DLL
----------------------------------------
DevExpress.XtraLayout.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraLayout.v21.2.DLL
----------------------------------------
OSPSuite.DataBinding
    Assembly Version: 3.0.0.4
    Win32 Version: 3.0.0.4
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.DataBinding.DLL
----------------------------------------
MoBi.Engine
    Assembly Version: 11.2.138.0
    Win32 Version: 11.2.138
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/MoBi.Engine.DLL
----------------------------------------
System.Runtime.Remoting
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Remoting/v4.0_4.0.0.0__b77a5c561934e089/System.Runtime.Remoting.dll
----------------------------------------
MoBi.Assets
    Assembly Version: 11.2.138.0
    Win32 Version: 11.2.138
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/MoBi.Assets.DLL
----------------------------------------
System.ComponentModel.DataAnnotations
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.ComponentModel.DataAnnotations/v4.0_4.0.0.0__31bf3856ad364e35/System.ComponentModel.DataAnnotations.dll
----------------------------------------
DynamicProxyGenAssembly2
    Assembly Version: 0.0.0.0
    Win32 Version: 4.3.1
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Castle.Core.dll
----------------------------------------
UIAutomationClient
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4614.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/UIAutomationClient/v4.0_4.0.0.0__31bf3856ad364e35/UIAutomationClient.dll
----------------------------------------
UIAutomationTypes
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4614.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/UIAutomationTypes/v4.0_4.0.0.0__31bf3856ad364e35/UIAutomationTypes.dll
----------------------------------------
OSPSuite.Serializer
    Assembly Version: 3.0.0.1
    Win32 Version: 3.0.0.1
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Serializer.DLL
----------------------------------------
System.Xml.Linq
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml.Linq/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll
----------------------------------------
OSPSuite.SimModel
    Assembly Version: 4.0.0.53
    Win32 Version: 4.0.0.53
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.SimModel.DLL
----------------------------------------
OSPSuite.FuncParser
    Assembly Version: 4.0.0.54
    Win32 Version: 4.0.0.54
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.FuncParser.DLL
----------------------------------------
OSPSuite.Infrastructure.Serialization
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Infrastructure.Serialization.DLL
----------------------------------------
OSPSuite.Infrastructure.Reporting
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Infrastructure.Reporting.DLL
----------------------------------------
NHibernate
    Assembly Version: 5.2.0.0
    Win32 Version: 5.2.5.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/NHibernate.DLL
----------------------------------------
FluentNHibernate
    Assembly Version: 2.1.2.0
    Win32 Version: 2.1.2.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/FluentNHibernate.DLL
----------------------------------------
OSPSuite.TeXReporting
    Assembly Version: 3.0.0.4
    Win32 Version: 3.0.0.4
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.TeXReporting.DLL
----------------------------------------
OSPSuite.Infrastructure.Export
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Infrastructure.Export.DLL
----------------------------------------
OSPSuite.Infrastructure
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Infrastructure.DLL
----------------------------------------
Microsoft.Extensions.Logging
    Assembly Version: 3.1.6.0
    Win32 Version: 3.100.620.31605
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Microsoft.Extensions.Logging.DLL
----------------------------------------
OSPSuite.Presentation.Serialization
    Assembly Version: 11.2.155.0
    Win32 Version: 11.2.155
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.Presentation.Serialization.DLL
----------------------------------------
System.Data.Linq
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Data.Linq/v4.0_4.0.0.0__b77a5c561934e089/System.Data.Linq.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
OSPSuite.DataBinding.DevExpress
    Assembly Version: 6.0.0.2
    Win32 Version: 6.0.0.2
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/OSPSuite.DataBinding.DevExpress.DLL
----------------------------------------
System.Web
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4494.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.Web/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Web.dll
----------------------------------------
DevExpress.Pdf.v21.2.Core
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.Pdf.v21.2.Core.DLL
----------------------------------------
DevExpress.Images.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.Images.v21.2.DLL
----------------------------------------
DevExpress.XtraPrinting.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.XtraPrinting.v21.2.DLL
----------------------------------------
DevExpress.BonusSkins.v21.2
    Assembly Version: 21.2.3.0
    Win32 Version: 21.2.3.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/DevExpress.BonusSkins.v21.2.DLL
----------------------------------------
Microsoft.GeneratedCode
    Assembly Version: 1.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Data.SQLite
    Assembly Version: 1.0.112.0
    Win32 Version: 1.0.112.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/System.Data.SQLite.DLL
----------------------------------------
System.Transactions
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.Transactions/v4.0_4.0.0.0__b77a5c561934e089/System.Transactions.dll
----------------------------------------
System.EnterpriseServices
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.EnterpriseServices/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.EnterpriseServices.dll
----------------------------------------
System.Numerics
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Numerics/v4.0_4.0.0.0__b77a5c561934e089/System.Numerics.dll
----------------------------------------
Iesi.Collections
    Assembly Version: 4.0.0.4000
    Win32 Version: 4.0.4.0
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Iesi.Collections.DLL
----------------------------------------
ICSharpCode.SharpZipLib
    Assembly Version: 1.3.2.10
    Win32 Version: 1.3.2.10
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/ICSharpCode.SharpZipLib.DLL
----------------------------------------
Northwoods.Go.Xml
    Assembly Version: 5.2.0.46
    Win32 Version: 5.2.0.46
    CodeBase: file:///C:/Program%20Files/Open%20Systems%20Pharmacology/MoBi%2011.2/Northwoods.Go.Xml.DLL
----------------------------------------

***** JIT Debugging *****
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.

**Comment by msevestre** [comment]
@AyaSaleh90 
Check this out 
https://github.com/Open-Systems-Pharmacology/Forum/discussions/1527#discussioncomment-5878325

**Reply by AyaSaleh90** [reply]
I have deleted this setting file and the project space worked again, thanks a lot :)

**Reply by AyaSaleh90** [reply]
Sorry but again, upon just opening one of PI steps, it crashed again gave me the previous error again and this one:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/67606842/16940a08-08b0-4eb8-9300-86c9d2aa9d69)

**Comment by msevestre** [comment]
as to 100+ obs data....what is the purpose of what you are trying to do with so many observed data points to fit at the same time?

**Reply by AyaSaleh90** [reply]
In short to have more confidence in the early estimated parameters using mean data, so we have individual data for a drug using different dosings with different phenotypes of the main enzyme metabolising the parent drug, with metabolite data as well. And I was running multiple PI steps at the same time with 2 different projects to save time since I know the steps already, but it kept crashing a couple of times.

**Reply by msevestre** [reply]
yes this is just too much. IT creates one tab per observed data. MoBi / PKsim is not made to handle this at the moment. We would need to reorganzie the logic so that only one tab is open at a time, and not ALL of them. Nothing we can do IMO at the moment besides rethinking your PI. Maybe using mean values instead of individual values would be sufficient

**Comment by Yuri05** [comment]
@AyaSaleh90 
>I tried increasing the value of GDIProcessHandleQuota, as suggested in one of the discussions

Did you set it to the **maximal possible value** and restarted your computer after this?
Max value is  65536 (decimal) or 10000(hex)

**Reply by AyaSaleh90** [reply]
@Yuri05  yes I set it to 1000 hexadecimal and still faced the same problem

**Reply by Yuri05** [reply]
1000 or 10000?

**Reply by AyaSaleh90** [reply]
@Yuri05 Sorry, it is 10000, isnt this the maximum value?

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/67606842/e3aa305c-6ef3-4e78-a198-f33109d85f3b)

**Reply by Yuri05** [reply]
Yes, this is the maximal value.
Can you start Windows Task manager, go to the details view, enable the columns **GDI objects** and **User objects**
Then start MoBi, open the problematic project and observe those values in the Task manager.
Which maximal values do you see for MoBi shortly before MoBi crashes?
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/a93f708a-d647-4a47-b768-63ce707c6833)
